package.json

```diff
@@ -50,8 +50,6 @@
     "typescript": "^5.4.2"
   },
   "engines": {
-    "pnpm": "^8.15.4",
     "node": ">=20.0.0"
-  },
-  "packageManager": "pnpm@8.15.5"
+  }
```

Comment:
We should keep the `engines` and `packageManager` in `package.json`

Reply:
We talked with Tadej, he added it there by mistake and all builds fail. In release version pnpm uses latest. So this should be removed